app/controllers/events_controller.rb

```diff
@@ -32,8 +33,13 @@ def index
       end
       format.text
       format.ics do
+        events = Event.all
```

Comment:
Would it be possible to put in a small test here? Just something that makes a few events and then checks the response is an ical feed? (maybe parse the result and check it's getting back the correct number of events?)

Reply:
done!